storage/revision.go

```diff
@@ -32,7 +37,7 @@ func (a revision) GreaterThan(b revision) bool {
 }
 
 func newRevBytes() []byte {
-	return make([]byte, 8+1+8)
+	return make([]byte, revBytesLen)
```

Comment:
addressed in https://github.com/yichengq/etcd/commit/1214f775197e7be7a2b7b337dc4d6f54ffd3deb8